pkg/platformconfig/platformconfig.go

```diff
@@ -104,6 +106,21 @@ func (config *Config) GetFunctionMetricSinks() (map[string]MetricSink, error) {
 	return config.getMetricSinks(config.Metrics.Functions)
 }
 
+func (config *Config) GetDefaultServiceType() v1.ServiceType {
+
+	// if not k8s, this field should be empty
+	if config.Kind == "local" {
+		return ""
+	}
+
+	switch config.KubeFunctionExposureMode {
```

Comment:
thought it makes sense to try to stick as much as possible to k8s service types
Nuclio is about to give the user the common features between k8s & docker. It is okay not to fully support it in docker, but what you're doing in this pr is putting the milestones for that to happen
Let's align `KubeFunctionExposureMode` types to be common between kube/local
e.g.
```golang
const(
  // in docker / k8s expose function's port on host
  KubeFunctionExposureModeExposed KubeFunctionExposureMode = "exposed"
  // in docker, available for containers within network, in k8s available within cluster
  KubeFunctionExposureModeInternal KubeFunctionExposureMode = "internal"
```


and on kube platform, you can map Exposure to service type